app/src/main/java/com/dessalines/thumbkey/utils/Types.kt

```diff
@@ -40,13 +40,34 @@ data class KeyboardC(
 data class KeyItemC(
     val center: KeyC,
     val swipes: Map<SwipeDirection, KeyC>? = null,
+    val left: KeyC? = null,
```

Comment:
Now that you added all these, you might as well get rid of the swipes map, as its deprecated in favor of all these.

Reply:
Whoops, missed that. Thank you for noticing!

I also updated the numeric keyboards that I somehow missed the first time.

Reply:
Sweet, thx!